content/blog/2020/trying-something-new.md

```diff
@@ -0,0 +1,39 @@
+Title: Trying something different this year
+Date: 2020-09-27 03:00
+Category: 2020
+Slug: trying-something-different
+Summary: Our plan to give attendees more opportunities to speak at PyCon India 2020.
+
+### Submissions this year
+
+As you know, PyCon India 2020 is going online this year. With the current situation, the team wanted to take the unique opportunity to make our conference bigger, more inclusive, and much more accessible to our attendees all over the world. We received an oustanding response to the event, with a total of 243 talks submissions under 12 primary domains.
+
+### Difficult decision to choose amongst so many remarkable proposals
+
+We got many excellent talks this year, but we had limited slots available for the conference. So, we had to reject a lot of great talks. The review team had to consider various factors while selecting the talks, including a proper mix of beginner and advanced topics, providing an opportunity for first-time speakers, diversity, etc.
```

Comment:
We received many excellent proposals this year, but due to limited slots available for the conference.